frame/mosaic/src/lib.rs

```diff
@@ -183,6 +188,21 @@ pub mod pallet {
 	#[allow(clippy::disallowed_types)]
 	pub type Nonce<T: Config> = StorageValue<_, u128, ValueQuery>;
 
+	/// Remote AMM IDs that exist (NetworkId, AmmId).
+	/// Note that this is actually a set that does bookkeeping of valid AmmIds.
```

Comment:
Nice issue for another library type, double hashset?

Reply:
Yes, that's exactly what I want.